### PR TITLE
Update gcs buckets 

### PR DESCRIPTION
**Pull request by taylrj**
This PR updates gcs buckets according to https://github.com/twreporter/kubernetes-config/pull/26.

### CODE REVIEW

File: config-for-docker-build.js [hunk 1]
```diff
@@ -16,7 +16,12 @@ exports.options = {
   'gcs config': {
     projectId: process.env.KEYSTONE_GCS_PROJECT_ID,
     keyFilename: __dirname + '/gcskeyfile.json',
-    bucket: process.env.KEYSTONE_GCS_BUCKET,
+    bucket: {
+      audio: process.env.KEYSTONE_GCS_AUDIO_BUCKET,
+      video: process.env.KEYSTONE_GCS_VIDEO_BUCKET,
+      rawPhoto: process.env.KEYSTONE_GCS_RAW_PHOTO_BUCKET,
+      resizedImage: process.env.KEYSTONE_GCS_PUBSUB_RESIZE_IMAGE_OUTPUT_BUCKET
```
Comment (babygoat) [inline config-for-docker-build.js:23]: How about `KEYSTONE_GCS_RESIZE_IMAGE_BUCKET`

**Comment by babygoat** [comment]
Except for the naming, others LGTM.

File: config-for-docker-build.js (hunk 1 above)
Comment (taylrj) [reply config-for-docker-build.js]: Updated.

**Comment by babygoat** [comment]
LGTM

**Comment by taylrj** [comment]
Clean rebase.

**Pull request merged**